api/src/wfl/jdbc.clj

```diff
@@ -134,9 +134,9 @@
   [binding & body]
   `(let [id#    (rand-int 10000)
          init# ~(second binding)]
-     (log/info "JDBC transaction" id# "started to" (format-db init#))
```

Comment:
What's trace logging?

Reply:
TL;DR - In Log4j, 2 levels below `INFO`, providing finer-grained information compared to `DEBUG`.

Log4j's log level hierarchy includes the following:
```
…
INFO
DEBUG
TRACE
…
```

But @rfricke-asymmetrik just pinged me to say that Log4j will be going away completely in his work, and with that we'll also see a change in [available log levels](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry).  We will lose `TRACE` but gain `NOTICE`:

```
…
DEBUG | (100) Debug or trace information.
INFO | (200) Routine information, such as ongoing status or performance.
NOTICE | (300) Normal but significant events, such as start up, shut down, or a configuration change.
…
```

My recommendation there will be to audit the categorization of existing Log4j `TRACE, DEBUG, INFO` logs to determine the new appropriate level.

For background why I downgraded this two levels: latest development has us spinning up transactions more frequently rather than passing around existing transactions, so the volume of these logs obscured useful information without adding value.  I stopped short of removing completely.

Reply:
Ah - @rfricke-asymmetrik maybe we can restore trace? That now seems useful. Sorry - I know I asked you to do that.

Reply:
Also I completely agree this information isn't very helpful. Downgrading was the right decision. Maybe we can make reading environment variables traces too!?

Reply:
> Also I completely agree this information isn't very helpful. Downgrading was the right decision. Maybe we can make reading environment variables traces too!?

Jinx :) can do.

Reply:
> Ah - @rfricke-asymmetrik maybe we can restore trace? That now seems useful. Sorry - I know I asked you to do that.

I actually don't mind trace going away as long as we commit to thoughtfully recategorizing existing logs below `INFO`.  Stackdriver's distinction between `INFO` and `NOTICE` is more instructive compared to "relatively how noisy do you want everything to be" haha

Reply:
Trace is not an option provided by Google Logging.  So even if we were to have a trace macro in the log namespace, it would just reference a different severity level like NOTICE or DEBUG.

Reply:
@rfricke-asymmetrik let's use NOTICE then :)

Reply:
That being said, there is a Trace field that can be added to the json log. So you can associate logs with a specific trace resource id from the Google Trace API.